test/perf/BUILD

```diff
@@ -55,6 +55,7 @@ syscall_test(
     size = "large",
     add_overlay = True,
     test = "//test/perf/linux:open_benchmark",
+    vfs2 = "True",
```

Comment:
Isn't vfs2 default enabled?

Reply:
Nope, its only used when the test specifies it.

Reply:
I saw in `test/runner/defs.bzl` around line 143 it is default true. If it is not, why it is only added in open_benchmark? Other tests don't need vfs2?

Reply:
Aaaaahhhh I see what happened, just a few days ago all the syscall tests made vfs2 the default. This PR started before that was true, updated it now.